bin/test-cleanup

```diff
@@ -24,6 +24,14 @@ if ! clusterroles=$(kubectl --context=$k8s_context get clusterroles -oname | gre
   echo "no clusterroles found for [$linkerd_namespace]" >&2
 fi
 
-if [[ $namespaces || $clusterrolebindings || $clusterroles ]]; then
-  kubectl --context=$k8s_context delete --wait=false $namespaces $clusterrolebindings $clusterroles
+if ! webhookconfigs=$(kubectl --context=$k8s_context get mutatingwebhookconfigurations -oname | grep -E  "/linkerd-$linkerd_namespace(-|$)"); then
```

Comment:
Not sure if this and the `if` in line 31 are necessary, since you are calling `kubectl delete` on all of them anyway at line 36. Do they just echo a warning?

Reply:
I added these lines to be consistent with the other cluster level resources and it appears to be done so that we log warnings if those resources aren't found.